test/topology_custom/conftest.py

```diff
@@ -1,9 +1,276 @@
 #
-# Copyright (C) 2023-present ScyllaDB
+# Copyright (C) 2022-present ScyllaDB
 #
 # SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
 #
 # This file configures pytest for all tests in this directory, and also
 # defines common test fixtures for all of them to use
+import pathlib
+import ssl
+import platform
+import time
+import urllib.parse
+from functools import partial
+from typing import List, Optional, Dict
+from test.pylib.random_tables import RandomTables
+from test.pylib.util import unique_name
+from test.pylib.manager_client import ManagerClient, IPAddress
+from test.pylib.async_cql import event_loop, run_async
+import logging
+import pytest
+from cassandra.auth import PlainTextAuthProvider                         # type: ignore # pylint: disable=no-name-in-module
+from cassandra.cluster import Session                                    # type: ignore # pylint: disable=no-name-in-module
+from cassandra.cluster import Cluster, ConsistencyLevel                  # type: ignore # pylint: disable=no-name-in-module
```

Comment:
Why not to merge it into the one line?
The same for imports from policies and connection

Reply:
this PR about migration, not optimizatin

https://github.com/scylladb/scylladb/blob/727d5637ab4e68943ddd9511d6c0cfb1e60c041a/test/topology/conftest.py#L21-L30